src/sif/pedestriancost.cc

```diff
@@ -540,7 +540,7 @@ PedestrianCost::PedestrianCost(const Costing costing, const Options& options)
   get_base_costs(costing_options);
 
   // Get the pedestrian type - enter as string and convert to enum
-  std::string type = costing_options.transport_type();
+  const const std::string&& type = costing_options.transport_type();
```

Comment:
This seems to be a bug in clang-tidy. Correct fix would be `const std::string&`.

Reply:
to me this again makes me think this isnt ready for prime-time. i wonder if we should let it run in CI and actually output the diff, but not fail the build when there is a diff. Then from time to time we can come back and make a PR like this where we look at the output and fix any bugs that it introduced. what do you think?